File: easy/019. The War of Problem Setters/Score.java
import java.util.Scanner;

class Score {
	public static void main(String[] argv) {
		Scanner input = new Scanner(System.in);
		String[] garry = input.nextLine().split(" ");
		String[] sharry = input.nextLine().split(" ");
		int garry_get = 0;
		int sharry_get = 0;

		for (int i = 0; i < garry.length; i++) {
			garry_get += Integer.parseInt(garry[i]);
			sharry_get += Integer.parseInt(sharry[i]);
		}

        if (garry_get == sharry_get) {
            System.out.println("None 0");
        } else {
            System.out.println(garry_get > sharry_get ? 
                               "Garry " + (garry_get - sharry_get) :
                               "Sharry " + (sharry_get - garry_get));
        }
	}
}

File: easy/019. The War of Problem Setters/score.py
garry = list(map(int, input().split()))
sharry = list(map(int, input().split()))
garry_get = 0
sharry_get = 0

for score in range(len(garry)):
    garry_get += garry[score]
    sharry_get += sharry[score]

if garry_get == sharry_get:
    print("None 0")
else:
    print("Garry " + str(garry_get - sharry_get) \
          if garry_get > sharry_get \
          else "Sharry " + str(sharry_get - garry_get))